index.html

```diff
@@ -1292,6 +1288,396 @@ <h3 id="set-permission-command">
+              Is [=policy-controlled feature=]?
+            </th>
+            <th rowspan="2">
+              Is [=powerful feature=]?
+            </th>
+            <th rowspan="2">
+              Specification
+            </th>
+            <th colspan="3">
+              Implementations
+            </th>
+          </tr>
+          <tr>
+            <th>
+              <a href="https://www.chromium.org/Home/">Chromium</a>
+            </th>
+            <th>
+              <a href="https://developer.mozilla.org/en-US/docs/Glossary/Gecko">Gecko</a>
+            </th>
+            <th>
+              <a href="https://webkit.org/">WebKit</a>
+            </th>
+          </tr>
+          <tr data-cite="geolocation">
+            <td class="string-token">
+              <a>"geolocation"</a>
+            </td>
+            <td class="is-policy-controlled">
+              YES
+            </td>
+            <td class="is-powerful-feature">
+              YES
+            </td>
+            <td class="spec">
+              [[[geolocation]]]
+            </td>
+            <td class="imp-chromium">
+              YES
+            </td>
+            <td class="imp-gecko">
+              YES
+            </td>
+            <td class="imp-webkit">
+              YES
+            </td>
+          </tr>
+          <tr data-cite="notifications">
+            <td class="string-token">
+              "[=notifications=]"
+            </td>
+            <td class="is-policy-controlled">
+              NO
+            </td>
+            <td class="is-powerful-feature">
+              YES
+            </td>
+            <td class="spec">
+              [[[NOTIFICATIONS]]]
+            </td>
+            <td class="imp-chromium">
+              YES
+            </td>
+            <td class="imp-gecko">
+              YES
+            </td>
+            <td class="imp-webkit">
+              YES
+            </td>
+          </tr>
+          <tr data-cite="push-api">
+            <td class="string-token">
+              [="push"=]
+            </td>
+            <td class="is-policy-controlled">
+              NO
+            </td>
+            <td class="is-powerful-feature">
+              YES
+            </td>
+            <td class="spec">
+              [[[push-api]]]
+            </td>
+            <td class="imp-chromium">
+              YES
+            </td>
+            <td class="imp-gecko">
+              YES
+            </td>
+            <td class="imp-webkit">
+              YES
+            </td>
+          </tr>
+          <tr data-cite="Web-share">
+            <td class="string-token">
+              <a>"web-share"</a>
+            </td>
+            <td class="is-policy-controlled">
+              YES
+            </td>
+            <td class="is-powerful-feature">
+              NO
+            </td>
+            <td class="spec">
+              [[[Web-Share]]]
+            </td>
+            <td class="imp-chromium">
+              NO
```

Comment:
I believe this might be a "YES" now, but it's just not supported on Desktop chrome:
https://wpt.fyi/results/web-share?label=experimental&label=master&aligned

(but Edge does support it on Desktop)

```suggestion
              YES
```